OrbitGl/CaptureWindow.cpp

```diff
@@ -675,6 +678,10 @@ void CaptureWindow::Draw() {
     glVertex3f(m_MouseX, m_WorldTopLeftY - m_WorldHeight, Z_VALUE_TEXT);
     glEnd();
   }
+  auto end = std::chrono::steady_clock::now();
+  auto dur = std::chrono::duration<double>(end - start);
+  total_draw_time_ += dur.count();
```

Comment:
What's the unit?

Reply:
Seconds. Changed the var name to reflect that, thanks.